vehicle-python/vehicle/__init__.py

```diff
@@ -54,7 +53,7 @@ def __init__(
         self.quantifier_sampling: Dict[str, Callable] = quantifier_sampling
         self.current_decl = None
 
-        self.debug: bool = False
+        self.debug: bool = True
```

Comment:
Please could you turn `debug` back to `False` so we don't get lots of output in the tests?